src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/PanelStatus.tsx

```diff
@@ -20,28 +20,32 @@ export function PanelStatus({
   analysisPoints,
   prefixBadge,
 }: {
-  analysisPoints: AnalysisPayload;
+  analysisPoints?: LocationAnalysisSummary;
   prefixBadge: PrefixBadge;
 }) {
   const isIndexed = useSelector(getIsIndexed);
   const time = useSelector(getCurrentTime);
   let status = "";
 
-  const points = analysisPoints?.data;
+  const points = analysisPoints?.data || [];
   const error = analysisPoints?.error;
 
   if (!isIndexed || !analysisPoints) {
```

Comment:
And then this line should change to be just `if (!points)`

Reply:
Lemme try this out